Free thread builds are failing ... not sure why .

@electroglyph maybe you can help.

For now... the plan it to disable free thread builds so that we can progress with Python 3.14

```
     Creating library build\temp.win-amd64-cpython-314t\Release\twisted_iocpsupport\iocpsupport.cp314t-win_amd64.lib and object build\temp.win-amd64-cpython-314t\Release\twisted_iocpsupport\iocpsupport.cp314t-win_amd64.exp
  iocpsupport.obj : error LNK2001: unresolved external symbol _PyFrame_SetLineNumber
  iocpsupport.obj : error LNK2001: unresolved external symbol PyInterpreterState_GetIDFromThreadState
  build\lib.win-amd64-cpython-314t\twisted_iocpsupport\iocpsupport.cp314t-win_amd64.pyd : fatal error LNK1120: 2 unresolved externals
  error: command 'C:\\Program Files\\Microsoft Visual Studio\\2022\\Enterprise\\VC\\Tools\\MSVC\\14.44.35207\\bin\\HostX86\\x64\\link.exe' failed with exit code 1120
```

--- a/CHANGELOG.rst
+++ b/CHANGELOG.rst
@@ -2,11 +2,11 @@ Changelog
 =========
 
 
-25.10.0 (2025-19-27)
+25.10.1 (2025-19-27)
 -------------------
 
 - Add support for Python 3.14 wheels.
-- Add support for Python 3.14 free threading wheels.
+- Free threading wheels builds were removed.
 
 
 25.2.0 (2025-02-27)

--- a/setup.cfg
+++ b/setup.cfg
@@ -1,6 +1,6 @@
 [metadata]
 name = twisted-iocpsupport
-version = 25.10.0
+version = 25.10.1
 description = An extension for use in the twisted I/O Completion Ports reactor.
 long_description = file: README.rst
 long_description_content_type = text/x-rst

